src/main/scala/beam/agentsim/infrastructure/charging/ChargingInquiryData.scala

```diff
@@ -0,0 +1,19 @@
+package beam.agentsim.infrastructure.charging
+import beam.agentsim.agents.choice.logit.MultinomialLogit
+
+case class ChargingInquiryData[A, T](utility: Option[MultinomialLogit[A, T]]) {
```

Comment:
Is this going to end up being used? Since @robfitzgerald said earlier this was a sketch, and it seems to still not be used. It is currently just passed around, but not utilized.

Reply:
As far as I know it is intended to be used later on in the ParkingSearch. We agreed to pass over the whole utility function to include it in the search as a constraint if more than one parking spot with charging capability is available and to ensure that we take into account personal preferences on charging while selecting the parking spot.

Reply:
this will be replaced with the work that Johannes is up to. it's wired into `beam.agentsim.agents.ChoosesParking` and was written by Johannes, so i left it in as a placeholder. it will not interfere with runs of BEAM using this PR and nothing will end up instantiating a ChargingInquiryData object (it's always Option[ChargingInquiryData] = None).

Reply:
this will be replaced with the work that Johannes is up to. it's wired into `beam.agentsim.agents.ChoosesParking` and was written by Johannes, so i left it in as a placeholder. it will not interfere with runs of BEAM using this PR and nothing will end up instantiating a ChargingInquiryData object (it's always `Option[ChargingInquiryData] = None`).

Reply:
ok, just wanted to make sure it didn't get lost in the shuffle :)